pyproject.toml

```diff
@@ -1,15 +1,17 @@
 [tool.black]
 line_length = 130
 target_version = ["py310"]
-force-exclude = ".*_pb2.pyi"
 skip_magic_trailing_comma = true
+# Exclude protobuf files because they have long line lengths
+# Ideally, we could configure black to allow longer line lengths
+# for just these files, but doesn't seem possible yet.
+force-exclude = ".*_pb2.pyi"
```

Comment:
I was not able to resolve this. It's very difficult to generate code that obeys line limits as such short of reimplementing black.

I did not want to make black a dependency of mypy-protobuf.

I may be able to run black as post-processing on the .pyi files as part of scripts/ - trying it out.

Reply:
Boom! Got that plan to work!

Now the flow for typeshed is to run the resulting generated protobuf code through black.
I will try my best to mostly make sure that the generated code is black-compatible, though line length constraint might be slightly different.

Reply:
argh - this failed because it changed lines like this
```python
class _LabelEnumTypeWrapper(google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[FieldDescriptorProto._Label.ValueType], builtins.type):  # noqa: F821
```

to 
```python
class _LabelEnumTypeWrapper(
    google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[FieldDescriptorProto._Label.ValueType], builtins.type
):  # noqa: F821
```

And the `# noqa` stopped working.

For context: the noqa is regarding a forward declaration that flake8 can't figure out, but mypy/pyright/python are fine with. The construct appears fine - but flake8 can't seem to handle it.

Just going to revert back to the working version of this PR - where we still ignore generated files from `black`.